src/workflows-app/SubmissionConfig.js

```diff
@@ -377,7 +389,27 @@ export const BaseSubmissionConfig = (
             ),
           ]),
         ]),
-        div({ style: { marginTop: '2rem', height: '2rem', fontWeight: 'bold' } }, ['Select a data table']),
+        div({ style: { marginTop: '1rem' } }, [
+          div({ style: { height: '2rem', marginRight: '0.25rem', fontWeight: 'bold', display: 'inline-block' } }, ['Call Caching:']),
+          div({ style: { display: 'inline-block', marginRight: '1rem' } }, [
+            h(InfoBox, [
+              "Call caching detects when a job has been run in the past so that it doesn't have to re-compute results. ",
+              h(Link, { href: getSupportLink('360047664872'), ...Utils.newTabLinkProps }, ['Click here to learn more.']),
+            ]),
+          ]),
+          div({ style: { display: 'inline-block' } }, [
+            h(Switch, {
```

Comment:
There is not a deployed PR site that I can use to try this out, but I imagine you will want to link "Call Caching" to the Switch as a label (for accessibility). See https://github.com/DataBiosphere/terra-ui/blob/39759fec9964e4ca3a80eb47d685a911b0f9c737/src/pages/workspaces/workspace/ShareWorkspaceModal/ShareWorkspaceModal.ts#L229 for an example.